docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
-could be a natural place to store the raw count matrix, but as long as the locations of the layers are specified, it can go anywhere.
-
-
-### Schema Version
-
-Datasets in the Data Portal must store the version of the schema they follow (that is, the version of this document) as
-well as the version of the particular encoding used. The encoding is documented
-[elsewhere](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) and describes techincal details
-of how the schema should be serialized in a particular file format.
-
-**Field name**|**Constraints**
-:--|:--
-corpora_schema_version|Follows [SemVer](https://semver.org/) conventions
-corpora_encoding_version|Follows [SemVer](https://semver.org/) conventions
+cellxgene's data requirements are tailored to optimize data reuse. Because each assay has different characteristics, our requirements differ by assay type. In general,
+cellxgene requires submission of "raw" data suitable for computational reuse, when a standard form exists, and strongly suggests that a "final" matrix suitable for
+visualization in the explorer be included. cellxgene uses `AnnData` for data ingestion, and uses `AnnData`'s `layers` functionality to accept multiple matrix layers,
+such as "raw" and "final". This imposes some requirements on data of all assay types:
+
+*   The "final" matrix MUST be stored in `AnnData.X`
+*   If a raw matrix is required for an assay type, it MUST be stored in `AnnData.layers["raw"]`.
+*   Additional layers provided at author discretion MAY be stored in `AnnData.layers` with author-selected keys.
+*   AnnData requires that matrix layers MUST have the same dimension, so raw count matrices MUST include the same cells and genes as the final. Because it is impractical to
+    retain all barcodes in raw and final matrices, cells MUST be filtered from both. By contrast, those wishing to reuse datasets require access to raw gene expression values,
+    so genes MUST be present in both datasets. Summarizing, any cell barcodes that are removed from the data MUST be filtered from both raw and final matrices.
+    By contrast, genes MUST NOT be filtered from the raw matrix. Any genes that publishers wish to filter from the final matrix MAY have their values replaced by np.nan,
+    which will mask them from exploration.
+
+In addition to these general requirements, the following table describes the matrix data and layers requirements that are assay-specific. If cellxgene does not support
+an assay you would like to publish, please post an issue on this repository to start a conversation about extending the schema. If an entry in the table is empty,
+the cellxgene schema does not have any other requirements on data in those layers beyond the ones listed above. This is usually the case when there are many ways to
+produce the matrix layer in question.
+
```

Comment:
I think that this table would be slightly easier to read with the following column order:

| Assay | "raw" Required? | "raw" requirements | Final ("X") Required? | Final ("X") Requirements | Other Layers |
|-------|-----------------|--------------------|-----------------------|--------------------------|--------------|

Reply:
Addressed in d04303b